application/routes/sftp.py

```
@@ -105,31 +114,38 @@
 
     status, reason = sftp.chmod(path, mode, recursive)
     if not status:
+        logger.error(f"Sftp: CHMOD failed: {reason}")
         abort(400, reason)
 
+    logger.info("Sftp: CHMOD successful")
     return 'success'
 
 
 @api.route('/sftp_mkdir/<session_id>', methods=['PUT'])
 def sftp_mkdir(session_id):
```

Ruff

Missing return type annotation for public function `sftp_mkdir`

Add return type annotation: `str`

(ANN201)